### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by naestia**
Update changelog to include changes made from tag 0.5.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,30 @@
+# 0.6.0
+
+## Prelude
+
+Added new features and commands, such as:
```
Comment (Grokzen) [inline CHANGELOG.md:5]: Remo `such as`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
+
+- Added error handling for 'subgit pull'.
```
Comment (Grokzen) [inline CHANGELOG.md:14]: Is it `Added` or `Improved`? was it there previous handling that we made better or not

File: CHANGELOG.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+	- If subgit config file has a repo with `branch: `, the command exits
+
```
Comment (Grokzen) [inline CHANGELOG.md:16]: Indent error? and generally there should not be blank lines as that would make a dent in the list and make two lists

File: CHANGELOG.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+- Implemented multiprocessing for 'subgit fetch' commnand. [#41][https://github.com/dynamist/subgit/pull/41]
+
+## New features
+
+* [#42][https://github.com/dynamist/subgit/pull/42] - Added function to render all repos from a github/gitlab user account/organisation
```
Comment (Grokzen) [inline CHANGELOG.md:21]: You need to update all links, the format for markdown links are `[<text>](url)` so you need to change the second `[ ]` to `( )`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1 +1,30 @@
...
+* [#41][https://github.com/dynamist/subgit/pull/41] - Implemented multiprocessing to 'subgit fetch' command
+* [#37][https://github.com/dynamist/subgit/pull/37] - Added '--conf' flag to use optional file name for subgit config
+* [#36][https://github.com/dynamist/subgit/pull/36] - Added 'subgit delete' command
+
+## Other notes
+
+* [#38][https://github.com/dynamist/subgit/pull/38] - Added a mailmap to repo
```
Comment (Grokzen) [inline CHANGELOG.md:28]: This could be put to `General fixes` section

File: CHANGELOG.md [hunk 6]
```diff
@@ -1 +1,30 @@
...
+## Prelude
+
+Added new features and commands, such as:
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
```
Comment (Grokzen) [inline CHANGELOG.md:12]: This could be renamed just `General` or `Fixes` or something less generic?

File: CHANGELOG.md (hunk 4 above)
Comment (Grokzen) [inline CHANGELOG.md:21]: The word `render` feels wrong context or purpose here. Try to find some other wording for this if possible.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1 +1,30 @@
...
+
+- Added error handling for 'subgit pull'.
+	- If subgit config file has a repo with `branch: `, the command exits
+
+- Implemented multiprocessing for 'subgit fetch' commnand. [#41][https://github.com/dynamist/subgit/pull/41]
```
Comment (Grokzen) [inline CHANGELOG.md:17]: You have this PR41 in both this general and new features. Each PR should only be in one block

**Review by Grokzen — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (naestia) [reply CHANGELOG.md:14]: I added a check before the main pull logic that wasn't there before

**Pull request merged**